scripts/infra/README.md

```diff
@@ -49,6 +50,9 @@ Either way, you'll need to edit the config to reference your cloud environment(s
 Example:
 
 ```bash
+# Ensure proper system packages are installed on a Red Hat-based system (Fedora/RHEL/CentOS)
```

Comment:
How about:

Ensure proper system packages are installed **if your local system is Red Hat-based**